html-web-page/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- your HTML code here 👇 -->
+<h1>Jelly of the Month Club</h1>
+<h2>Make all your friends jelly of your stock of jelly!</h2>
+
+<p>Here at JOTMC Inc., we are constantly striving to supply our customers with the highest-quality and most unique jellies on a monthly basis. For the jelly lover in your life (or yourself!), this is the gift that keeps on giving all year round. We offer three tiers of monthly jelly joy, and each one offers their own perks. Here is a breakdown of each, as well as your monthly price:</p>
+
+<div>
+  <table>
+    <thead>
+    <tr>
+      <th></th>
+      <th scope="col">Basic Plan</th>
+      <th scope="col">Deluxe Plan</th>
+      <th scope="col">Jelly Giant Plan</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <th scope="row">Three new curated jellies per month</th>
```

Comment:
Each of these first column elements should be TDs, you don't want the text centered as a heading.

Reply:
<img width="931" alt="Screen Shot 2021-10-04 at 5 24 01 PM" src="https://user-images.githubusercontent.com/86754302/135941719-f7ac0b23-478e-4175-9963-eb39376a88af.png">